disnake/__main__.py

```diff
@@ -271,10 +292,20 @@ def add_newbot_args(subparser):
     parser.add_argument(
         "directory", help="the directory to place it in (default: .)", nargs="?", default=Path.cwd()
     )
-    parser.add_argument(
+
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
         "--prefix", help="the bot prefix (default: $)", default="$", metavar="<prefix>"
     )
-    parser.add_argument("--sharded", help="whether to use AutoShardedBot", action="store_true")
+    group.add_argument(
+        "--app-commands-only",
+        help="whether to support prefix commands at all",
```

Comment:
This is the wrong way around, I can't think of the right phrasing though

Reply:
https://github.com/DisnakeDev/disnake/pull/791/commits/7ba2b9f4a89fb8bf8c6644804882119a17fabb06